test/e2e/specs/support/support__popover-valid.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * @group quarantined
+ * @group help-centre
```

Comment:
```suggestion
 * @group help-center
```

Reply:
I would not suggest making this change, because the TeamCity configuration is configured to pick up "help-centre" and as a Canadian I've used the Canadian/British English spelling for "center".